engines/wintermute/base/gfx/base_image.cpp

```diff
@@ -97,7 +97,7 @@ bool BaseImage::loadFile(const Common::String &filename) {
 	//
 	if (_filename.hasSuffix(".tga")) {
 		_paletteTga = new byte[_paletteCount * 3];
-		for (uint32 i = 0; i < _paletteCount * 3; i += 3) {
```

Comment:
casting should be avoided

Reply:
It may be simpler to change i type to int as _paletteCount is promoted to int before being multiplied by 3.
This is guaranteed to not overflow.

Reply:
This code is now deprecated after commit a3dd9b15fe4153a0c1bb7aceb693e12abd2408da